### ISSUE TITLE
Mac install won't launch

### ISSUE THREAD

**Issue by Jertherrien** [ImpulseAdventure/GUIslice-Builder#274]
When i trie to launch the files so the app will open it just won't.
I tried downloading the two tar file from GitHub. I double click on it to ''unzip'' it. Then I one the GUISLICE folder. and I can't fin the JAR file to double click on it. 

I tried to double click on every file in the folder but nothing will launch the guislice app. 
Tried to download Java from oracle but it still won't launch any of the file in the GUISLICDE folder.

Il somebody with a Mac Sonoma Version 14.3.1 can do a video of the exact procedure it would be very appreciated. 
Thanks.

![Uploading 1000000590.jpg…]()

**Comment by Pconti31** [comment]
@Jertherrien Did you follow the [readme ](https://github.com/ImpulseAdventure/GUIslice-Builder/blob/master/README.md)?
Open a shell.
cd to the GUIsliceBuilder folder.
./GUIslice.sh

There is no jar file since just like adruino ide java is built into the Builder app.
Paul--

**Comment by Jertherrien** [comment]
Finaly I was able to solve the problem. It may be my lack of Mac experience.
If there is another person with the same problem here is what I did to solve my problem :
1: Download the program from the GitHub for your Mac.
2: Go into your folder and slide the GUIslicebuilder folder to the application folder. 
3: Open the app called : terminal
4: Write the letters : ls    then press return
5: You now see the general folder of your computer.
6: Write: cd applications     then press return
7:  write the letters : ls    then press return
8:  write: cd GUIslicebuilder         or whatever the name of the folder that you downloaded from GitHub then press return
9: write:   ./GUIslice.sh
10: voila you did it
